### PR TITLE
WIP - mock glbc-controller endpoint

### PR DESCRIPTION
**Pull request by roivaz**

### CODE REVIEW

File: utils/local-setup.sh [hunk 1]
```diff
@@ -61,8 +61,12 @@ set -e pipefail
 trap cleanup EXIT 1 2 3 6 15
 
 cleanup() {
-  echo "Killing KCP"
-  kill "$KCP_PID"
+  if [[ -n ${LOCAL_SETUP_BACKGROUND} ]] then
+    echo "Exiting. Use 'pkill kcp' to shutdown kcp"
```
Comment (david-martin) [inline utils/local-setup.sh]:
```suggestion
  if [[ -n ${LOCAL_SETUP_BACKGROUND} ]]
  then
    echo "Exiting. Use 'pkill kcp' to shutdown kcp"
```
This gave an error for me locally `./utils/local-setup.sh: line 64: syntax error near unexpected token `then'`

Comment (roivaz) [reply utils/local-setup.sh]: right, I don't know how it didn't fail for me

**Comment by roivaz** [comment]
@david-martin I will squash all the commits before merging as the history is extremely dirty because I tried different things until it all worked.

**Comment by roivaz** [comment]
@david-martin added some more commits to this branch. Now the plugin is able to get the live state of the application resources from argocd API. See https://github.com/roivaz/kcp-glbc/pull/1 for the specific change for this.

**Pull request closed without merging**